### PR TITLE
Rest server job model refactor

### PR DESCRIPTION
**Pull request by abuccts**
Rest server job model refactor.

### CODE REVIEW

File: src/rest-server/yarn.lock [hunk 1]
```diff
@@ -2420,6 +2420,11 @@ mocha@~5.0.0:
     mkdirp "0.5.1"
     supports-color "4.4.0"
 
+module-alias@^2.2.0:
```
Comment (SparkSnail) [inline src/rest-server/yarn.lock]: does this file need to be changed?

File: src/rest-server/package.json [hunk 2]
```diff
@@ -60,6 +61,15 @@
     "winston": "~2.4.0",
     "xml2js": "~0.4.19"
   },
+  "_moduleAliases": {
+    "@config": "src/config",
```
Comment (MatsukawaTadashi) [inline src/rest-server/package.json]: why not create a alias of root dir `src/`

Comment (sunqinzheng) [inline src/rest-server/package.json]: Why not create an alias of root directory `src/`?

Comment (sunqinzheng) [reply src/rest-server/package.json]: P.S. webpack also has a built-in module alias feature https://webpack.js.org/configuration/resolve/#resolvealias

File: src/rest-server/yarn.lock (hunk 1 above)
Comment (abuccts) [reply src/rest-server/yarn.lock]: yes, added a new package `module-alias` in `package.json`

File: src/rest-server/package.json (hunk 2 above)
Comment (abuccts) [reply src/rest-server/package.json]: it's OK to create an alias for `src` but the path will be longer?

we don't use `webpack` in rest server

Comment (Gerhut) [reply src/rest-server/package.json]: Have you tried `NODE_PATH` environment variable?

Comment (abuccts) [reply src/rest-server/package.json]: > Have you tried `NODE_PATH` environment variable?

seems it does not work on windows using dotenv or command inline env

File: src/rest-server/src/models/user.js [hunk 3]
```diff
@@ -259,7 +259,7 @@ if (config.env !== 'test') {
         if (errMsg) {
           logger.warn('get user list failed', errMsg);
         } else {
-          logger.warn('users:', userInfoList);
+          // logger.warn('users:', userInfoList);
```
Comment (SparkSnail) [inline src/rest-server/src/models/user.js]: why annotate this?

Comment (abuccts) [reply src/rest-server/src/models/user.js]: do not need to list all users in log, seems it was a debug log

**Pull request merged**